.github/workflows/linting.yml

```diff
@@ -14,9 +14,9 @@ jobs:
   EditorConfig:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v3
```

Comment:
These look odd, since a v4 went to v3 -- problem from creating the branch for this from the wrong starting branch?

Reply:
I had some linting errors so I ran ´nf-core lint --fix files_unchanged´

Reply:
With the most up to date tools? What's in the files under review for release? I suppose we want to be identical to those.